apps/docs-website/docs/core-philosophy/stylistic-choices.md

```diff
@@ -8,32 +8,33 @@ description: 🔥 Hot takes!
 
 ## No reduce
 
 - [Is reduce() bad? - HTTP 203](https://www.youtube.com/watch?v=qaGjS7-qWzg)
-- [TkDodo - Why i don't like reduce](https://www.youtube.com/watch?v=qaGjS7-qWzg)
+- [Why I don't like reduce | TkDodo's Blog](https://tkdodo.eu/blog/why-i-dont-like-reduce)
 
 :::info
 
-Sheriff actually allows using `reduce` for very simple operations like summing up numbers. Complex operations are banned.
+Sheriff actually allows using `reduce` for basic operations like summing up numbers.
+Only complex operations are banned.
 
 :::
 
 ## No enums
 
-- [eslint-plugin-typescript-enum - README](https://github.com/shian15810/eslint-plugin-typescript-enum/blob/main/README.md)
+- [eslint-plugin-typescript-enum](https://github.com/shian15810/eslint-plugin-typescript-enum#readme)
 - [Enums considered harmful](https://www.youtube.com/watch?v=jjMbPt_H3RQ)
 - [How to use TypeScript Enums and why not to, maybe](https://www.youtube.com/watch?v=pWPClHdcvVg)
 - [Let's Talk About TypeScript's Worst Feature](https://www.youtube.com/watch?v=Anu8vHXsavo)
-- [Effective Typescript book](https://effectivetypescript.com) explicitly warn against this, saying that `enums` belong to a legacy design choice of Typescript.
+- [Effective TypeScript](https://effectivetypescript.com) explicitly warn against this, saying that `enums` belong to a legacy design choice of TypeScript.
 
 ## No overloads
 
-- generics supersedes them. Overloads are a legacy feature that was made available in Typescript before generics were a thing. Overloads are mostly a C#/Angular leftover. Simply put: there are no problems that function overloads solve better than generics
-- overloads clutter the code and make it more verbose and harder to read, which increase the cognitive overload
-- overloads clutter the VScode tooltips
-- overloads force you to write non-standard javascript syntax
-- overloads are a bad practice. The flexibility that they enable also enable your team to write inconsistent code. Which is exactly the problem that ESLint is designed to solve
-- [Effective Typescript book](https://effectivetypescript.com) explicitly warn against this, saying that "conditional types" are preferrable.
+- Generics supersede them. Overloads are a legacy feature that was made available in TypeScript before generics were a thing. Overloads are mostly a C#/Angular leftover. Simply put: there are no problems that function overloads solve better than generics.
+- Overloads clutter the code and make it more verbose and harder to read, which increase the cognitive overload
+- Overloads clutter IDE tooltips.
+- Overloads force you to write non-standard JavaScript syntax.
+- The flexibility that overloads enable also enables your team to write inconsistent code—exactly the problem that ESLint is designed to solve.
+- [Effective TypeScript](https://effectivetypescript.com) explicitly warns against this, stating that conditional types are preferable.
```

Comment:
👎

Reply:
To uppercasing and periods, not to the phrasing changes?

Reply:
to everything